react/features/file-sharing/utils.ts

```diff
@@ -0,0 +1,27 @@
+const generateDownloadUrl = async (url: string) => {
+    const resp = await fetch(url);
+    const respBlob = await resp.blob();
```

Comment:
If we ever enlarge the file size this may become problematic because it downloads the content to memory first.

Reply:
that is true :(